rlw-examples/iis-monitoring.ps1

```diff
@@ -0,0 +1,5 @@
+# ---
+# RightScript Name: SYS IIS monitoring install
+# Description: Installs IIS specific monitoring.
+#
+rsc rl10 create /rll/tss/exec/iis_monitor executable=[System.IO.Path]::GetFullPath(".\attachments\iis-monitor.ps1")
```

Comment:
How will this work as a RightScript without `${env:RS_ATTACH_DIR}`? There should be examples using that elsewhere in this repository that you can look at for how to handle that so it also supports the cookbook method although the cookbook method isn't really used any more.